docs/arcc.md

```diff
@@ -0,0 +1,104 @@
+# nf-core/configs: ARCC Configuration
+
+The [Advanced Research Computing Center (ARCC)](http://www.uwyo.edu/arcc/) for the University
+of Wyoming (UW) has been set up to allow its users to utilize Nextflow with Singularity.
+
+## Getting Started
+
+First, you will need an account on ARCC, if you already have an account, skip ahead, otherwise
+please continue. To get an account, you will need to be a Principal Investigator (PI) or student
+at UW, or be sponsored by a UW PI. To learn more please visit [ARCC - HPC Account Requests](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1913684148/Accounts+Access+and+Security).
+
+With an account in hand, you are ready to proceed.
+
+## Running Nextflow
+
+When using Nextflow on ARCC it is recommended you launch Nextflow on one of the compute nodes,
+instead of the login nodes. To do this you will use the `salloc` command to launch an
+[Interactive Job](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1599078403/Start+Processing#Interactive-Jobs).
+
+Alongside `salloc`, please consider making use of [screen or tmux](https://arccwiki.atlassian.net/wiki/spaces/DOCUMENTAT/pages/1617494076/Screen+and+Tmux+Commands)
+in order to not lose your Interactive Job.
+
+Once you are on a compute node, you can then use the `module` command to load Conda and
+Singularity.
+
+### Creating a Nextflow environment
+
+As an ARCC user you may have noticed there is already a module for Nextflow. However, it
+may be out of date or limited to a single version. All nf-core pipelines have minimum Nextflow
+version requirements, so its easier to create a Nextflow environment, as it will ensure you
+have the latest available Nextflow version.
+
+```{bash}
+module load miniforge
+conda create -n nextflow -c conda-forge -c bioconda nextflow
+```
+
+## Parameters of Interest
+
+#### `--slurm_account`
```

Comment:
I'm not sure about the use of these custom parameters. They may result in red herring warnings in both nextflow and nf-core pipelines as they aren't in the code itself/schema. Although it's probably solved by adding to the ignore parameters, you have the risk of someone specifies their own custom config to ignore other parameters, you would lose your ignore list.

We've recommended to others to ask users to specify bash env variables with the information, and pull these in the config with the groovy function I've forgotten the name of now... But I'm pretty sure @nvnieuwk and @matthdsm use in the VSC Ghent profile (?) 

P.s. it's 5am and I'm on my phone thus the less than ideal referencing 😅

Reply:
I've seen mixed usage here. I think I could probably get away with a `SBATCH_ACCOUNT` env variable to be consistent with other configs